src/Microsoft.Health.Fhir.Anonymizer.Shared.Core/Utility/PresidioUtility.cs

```diff
@@ -0,0 +1,16 @@
+﻿
+using Microsoft.Health.Fhir.Anonymizer.Core.Api;
+
+namespace Microsoft.Health.Fhir.Anonymizer.Core.Utility
+{
+    public class PresidioUtility
+
+    {        
+        public static string Anonymize(string text, IApiHandler apiHandler)
```

Comment:
NIT: maybe AnalyzeAndAnonymize since this function is not only anonymizing.

Reply:
@SharonHart

Reply:
The idea was that the client doesn't have to know about the internal division we did, there is an overload of Anonymize methods and if just the text is passed (w/o analyzer results) it will fetch analyzer results itself.

Reply:
Lets leave it as is (Anonymize)